Demo/src/androidTest/java/com/braintreepayments/demo/test/utilities/TestHelper.java

```diff
@@ -74,21 +63,13 @@ public void setMerchantAccountId(String merchantAccountId) {
                 .edit()
                 .putString("merchant_account", merchantAccountId)
                 .commit();
-
-        SystemClock.sleep(2000);
-        onDevice(withText("Reset")).perform(click());
     }
 
     public void useTokenizationKey() {
         PreferenceManager.getDefaultSharedPreferences(ApplicationProvider.getApplicationContext())
                 .edit()
                 .putBoolean("tokenization_key", true)
                 .commit();
-
-        // additional sleep here makes tests more consistent for some reason
```

Comment:
The tests have been passing consistently without all these sleeps because we no longer need to call reset when we don't launch the app until after the settings have been configured. I think it's ok to remove all these now, but can add them back if the tests start to flake again.

Reply:
🙌